### PR TITLE
Add new `kafka_info` lib

### PR DESCRIPTION
**Pull request by StephenSorriaux**
## Proposed Changes

  - add a new `kafka_info` lib
  - get information about topics
```
{
    "results": {
        "test_1600292339": {
            "0": {
                "isr": [
                    1002
                ],
                "leader": 1002,
                "replicas": [
                    1002
                ]
            }
        }
    }
}
```
  - get information about brokers
```
{    
    "results": {
        "1001": {
            "host": "172.17.0.9",
            "nodeId": 1001,
            "port": 9092,
            "rack": null
        },
        "1002": {
            "host": "172.17.0.10",
            "nodeId": 1002,
            "port": 9092,
            "rack": null
        }
    }
}
```
  - get information about consumer groups
```
{
    "results": {
        "AWESOME_consumer_group_1607465801": {
            "coordinator": {
                "host": "172.17.0.9",
                "nodeId": 1001,
                "port": 9092,
                "rack": null
            },
            "error_code": 0,
            "group_state": "Empty",
            "members": {},
            "protocol": "",
            "protocol_type": "consumer"
        },
        "AWESOME_consumer_group_1607466258": {
            "coordinator": {
                "host": "172.17.0.10",
                "nodeId": 1002,
                "port": 9092,
                "rack": null
            },
            "error_code": 0,
            "group_state": "Stable",
            "members": {
                "kafka-python-2.0.1-e5500fee-8df9-4f37-bcd7-788522a1c382": {
                    "client_host": "/172.17.0.1",
                    "client_id": "kafka-python-2.0.1",
                    "member_assignment": {
                        "assignment": {
                            "test_1607465755": [
                                0
                            ]
                        },
                        "user_data": "",
                        "version": 0
                    },
                    "member_metadata": {
                        "subscription": [
                            "test_1607465755"
                        ],
                        "user_data": "",
                        "version": 0
                    }
                }
            },
            "protocol": "range",
            "protocol_type": "consumer"
        }
    }
}
```
  - add some tests in molecule
  - a lot of refacto
## TODO

- [x] documentation

- [x]  some refacto

### CODE REVIEW

**Comment by ryarnyah** [comment]
I will look at it next week if it's ok with you?

**Comment by StephenSorriaux** [comment]
Sure, thank you!

File: module_utils/kafka_manager.py [hunk 1]
```diff
@@ -827,11 +662,254 @@ def update_topic_assignment(self, json_assignment, zknode):
...
+                                'version': assign.version,
+                                'assignment': assignment,
+                                'user_data': assign.user_data.decode('utf-8')
+                            }
+                        }
+                    group = {
+                        'error_code': err,
+                        'group_state': gstate,
+                        'members': members,
+                        'coordinator': broker.nodeId
```
Comment (ryarnyah) [inline module_utils/kafka_manager.py:762]: `broker.nodeId` may not be human readable, might add `host` var?

File: module_utils/kafka_manager.py [hunk 2]
```diff
@@ -827,11 +662,254 @@ def update_topic_assignment(self, json_assignment, zknode):
...
+                                'user_data': mdata.user_data.decode('utf-8')
+                            },
+                            'member_assignment': {
+                                'version': assign.version,
+                                'assignment': assignment,
+                                'user_data': assign.user_data.decode('utf-8')
+                            }
+                        }
+                    group = {
+                        'error_code': err,
```
Comment (ryarnyah) [inline module_utils/kafka_manager.py:759]: May add `protocolType`.

File: module_utils/kafka_manager.py (hunk 1 above)
Comment (StephenSorriaux) [reply module_utils/kafka_manager.py]: You're right, I changed `coordinator` to be a dict with more information

File: module_utils/kafka_manager.py (hunk 2 above)
Comment (StephenSorriaux) [reply module_utils/kafka_manager.py]: Thanks! Added `protocol_type` and `protocol`, because I did not want to make any jealous :)

**Comment by ryarnyah** [comment]
LGTM 🍺

**Pull request merged**